app/org/sagebionetworks/bridge/play/controllers/SurveyController.java

```diff
@@ -178,9 +193,12 @@ public Result deleteSurvey(String surveyGuid, String createdOnString) throws Exc
         Survey survey = surveyService.getSurvey(keys);
         verifySurveyIsInStudy(session, studyId, survey);
         
-        surveyService.deleteSurvey(survey);
+        if ("true".equals(physical) && user.isInRole(Roles.ADMIN)) {
+            surveyService.deleteSurveyPermanently(survey);
```

Comment:
We've checked above and they have to be a developer or an admin, so if they're not an admin here, or don't ask for a physical delete, they fall back to logical delete.